src/validate.ts

```diff
@@ -76,13 +76,19 @@ export function validate(
 
   function newError(
     invalidNode: typescript.Node,
-    code: number,
-    messageText: string
+    error: ErrorCode,
+    messageText?: string
   ): ts.Diagnostic {
+    const baseUrl = process.env.FUNCTIONLESS_LOCAL
+      ? `http://localhost:3000`
+      : `https://functionless.org`;
```

Comment:
Inject the base url instead of the flag?

Reply:
Then every caller would have to pass it?